mlflow/store/model_registry/databricks_workspace_model_registry_rest_store.py

```diff
@@ -35,3 +44,69 @@ def _await_model_version_creation(self, mv, await_creation_for):
             "(https://docs.databricks.com/en/machine-learning/manage-model-lifecycle/index.html)."
         )
         self._await_model_version_creation_impl(mv, await_creation_for, hint=uc_hint)
+
+    def copy_model_version(self, src_mv, dst_name):
+        """
+        Copy a model version from one registered model to another as a new model version.
+        This method can be used within the Databricks workspace registry to copy model versions
+        between registered models, or to migrate model versions from the Databricks workspace
+        registry to Unity Catalog. During the migration, signature validation can be bypassed
+        by setting the `MLFLOW_SKIP_SIGNATURE_CHECK_FOR_MIGRATION_TO_DATABRICKS_UC_REGISTRY`
+        environment variable to `True`.
+
+        Args:
+            src_mv: A :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+                the source model version.
+            dst_name: The name of the registered model to copy the model version to. If a
+                registered model with this name does not exist, it will be created.
+
+        Returns:
+            Single :py:class:`mlflow.entities.model_registry.ModelVersion` object representing
+            the cloned model version.
+        """
+        if dst_name.count(".") == 2:
+            import mlflow
+            from mlflow.store._unity_catalog.registry.rest_store import (
+                UcModelRegistryStore,
+            )
+
```

Comment:
can we move this to the top level?

Reply:
I had it there since the other paths wouldn't need the import, so to prevent creating unnecessary dependencies. But if that breaks style guidelines, moved it up instead